### PR TITLE
Updates `upload-custom-translations` to support custom locales

### PR DESCRIPTION
**Pull request by dianabarsan**
When adding a new language, instead of falling back on using `values`, decide which translation format to use depending on app version. 

medic/medic#5617
medic/medic-conf#199

### CODE REVIEW

**Comment by dianabarsan** [comment]
@garethbowen please review.
Wondering if we should track this as a separate issue? 
Thanks!

File: src/fn/upload-custom-translations.js [hunk 1]
```diff
@@ -10,6 +12,7 @@ module.exports = (projectDir, couchUrl) => {
 
   const dir = `${projectDir}/translations`;
   const db = pouch(couchUrl);
+  const instanceUrl = couchUrl.replace(/\/medic$/, '');
```
Comment (garethbowen) [inline src/fn/upload-custom-translations.js]: This doesn't work when the db name isn't `medic`. In api/src/environment.js we do this...

```js
  const parsedUrl = url.parse(couchUrl.replace(/\/$/, ''));
  const instanceUrl = couchUrl.slice(0, couchUrl.lastIndexOf('/')),
```

Maybe we should have an `environment` module for medic-conf too?

Comment (dianabarsan) [reply src/fn/upload-custom-translations.js]: The whole of medic-conf doesn't work with db names that aren't `medic`:
https://github.com/medic/medic-conf/blob/master/src/lib/main.js#L171

I thought that would be part of a separate issue. Would you like me to include an overall update here?

Comment (garethbowen) [reply src/fn/upload-custom-translations.js]: Oh dear... Please create a separate issue to cover that.

**Pull request merged**